docs/source/notes/autograd.rst

```diff
@@ -91,6 +91,8 @@ the module level with :meth:`nn.Module.requires_grad_()`.
 When applied to a module, ``.requires_grad_()`` takes effect on all
 of the module's parameters (which have ``requires_grad=True`` by default).
 
+Tensors that are not module parameters have ``requires_grad=False` by default.
```

Comment:
Maybe this should be just after the paragraph line 61 mentioning that all the Tensor factory Functions set it to False by default and it can be changed via the `requires_grad` keyword argument.
Also mention that when a Tensor is wrapped inside a `nn.Parameter`, then its `requires_grad` field is automatically set to True.

Reply:
yeah i like that location better too. This is so far down I worry that no one would ever see it.

Reply:
what do you think of this?